README.md

```diff
@@ -33,6 +33,6 @@ as possible:
 
 The following chart gives an impression of the speed improvement that can be expected
 when processing larger files. The [benchmarks](https://github.com/geofileops/geobenchmark)
-ran on a Windows PC with 12 cores and include I/O.
+ran on a Windows PC with 12 cores and include I/O.  The benchmark dataset was 500K polygons.
```

Comment:
```suggestion
typically use input file(s) with 500K polygons, ran on a Windows PC with 12 cores and include I/O.
```

Reply:
True, is indeed important info to be able to interprete the timings.

Not sure what is best/clearest, but I tried to clarify that for most 2-layer operations (intersection, union) both input layers are 500k polygons. 

Note that the symmetrical difference benchmark has one 500k input layer, and one with only 10 features, but quite complex ones (20.000 points per polygon).

Reply:
Makes sense.